zer0-os/zOS
# feat: trigger verify id dialog from the messenger lists user header

### What does this do?
- opens the verify id dialog from the messenger lists user header `Verify ID` button.

### Why are we making this change?
- to provide more information to the user by opening a modal containing more information on verifying their id.

### How do I test this?
- log in with a user via a wallet, and that does not have a primaryZID, enable verify id feature flag and click the `Verify ID` button.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?




https://github.com/zer0-os/zOS/assets/39112648/35b1023b-7742-48d1-a8b1-cf2931de25ad
